astroquery/esa_hubble/core.py

```diff
@@ -0,0 +1,396 @@
+
+
+class ESAHubbleHandler(BaseQuery):
+
+    def __init__(self):
+        super(ESAHubbleHandler, self).__init__()
+
+    def get_file(self, filename, response, verbose=False):
+        file = open(filename, 'wb')
+        file.write(response.content)
+        file.close()
+
+        if "/" not in filename:
+            log.info("File " + str(filename) + " downloaded to current "
+                     "directory")
+        else:
+            log.info("File " + str(filename) + " downloaded")
+
+    def get_table(self, filename, response, output_format='votable',
+                  verbose=False):
+        file = open(filename, 'wb')
+        file.write(response.content)
+        file.close()
+
+        table = modelutils.read_results_table_from_file(filename,
+                                                        str(output_format))
+        return table
+
+    def request(self, t="GET", link=None, params=None):
+        return self._request(method=t, url=link, params=params)
+
+
+Handler = ESAHubbleHandler()
+
+
+class ESAHubbleClass(BaseQuery):
+
+    data_url = conf.DATA_ACTION
+    metadata_url = conf.METADATA_ACTION
+
+    def __init__(self, url_handler=None, tap_handler=None):
+        super(ESAHubbleClass, self).__init__()
+        if url_handler is None:
+            self._handler = Handler
+        else:
+            self._handler = url_handler
+
+        if tap_handler is None:
+            self._tap = TapPlus(url="http://hst.esac.esa.int"
+                                    "/tap-server/tap/")
+        else:
+            self._tap = tap_handler
+
+    def get_product(self, observation_id, calibration_level="RAW",
+                    filename=None, verbose=False):
+        """ Download products from EHST
+
+            Parameters
+            ----------
+            observation_id : string, id of the observation to be downloaded,
+            mandatory
+            The identifier of the observation we want to retrieve, regardless
+            of whether it is simple or composite.
+            calibration_level : string, calibration level, optional, default
+            'RAW'
+            The identifier of the data reduction/processing applied to the
+            data. By default, the most scientifically relevant level will be
+            chosen. RAW, CALIBRATED, PRODUCT or AUXILIARY
+            filename : string, file name to be used to store the artifact,
+            optional, default None
+            File name for the observation.
+            verbose : bool, optional, default 'False'
+            flag to display information about the process
+
+            Returns
+            -------
+            None. It downloads the observation indicated
+        """
+
+        obs_id = "OBSERVATION_ID=" + observation_id
+        cal_level = "CALIBRATION_LEVEL=" + calibration_level
+        link = self.data_url + obs_id + "&" + cal_level
+        result = self._handler.request('GET', link, params=None)
```

Comment:
`params=None` is redundant
```suggestion
        result = self._handler.request('GET', link)
```